### PR TITLE
AB-93: Update sequences in the db after an import of some data dump

### PR DESCRIPTION
**Pull request by paramsingh**
Also, add a command to manage.py to update_sequences, if needed.

### CODE REVIEW

File: db/dump.py [hunk 1]
```diff
@@ -238,6 +241,56 @@ def generate_where(row_name, start_t=start_time, end_t=end_time):
...
+            SELECT setval('{seq_name}', max(id))
+              FROM {table_name}
+        """.format(seq_name=seq_name, table_name=table_name)))
+
+
+def update_sequences():
+    """ Update all sequences to the maximum value of id in the table.
+    """
+    # lowlevel_id_seq
+    logging.info('Updating lowlevel_id_seq...')
```
Comment (alastair) [inline db/dump.py]: I've created https://tickets.metabrainz.org/projects/AB/issues/AB-364 to add some docs to remind us that we have to update this every time we make a new table.

File: db/dump.py [hunk 2]
```diff
@@ -238,6 +241,56 @@ def generate_where(row_name, start_t=start_time, end_t=end_time):
...
+    logging.info('Updating user_id_seq...')
+    update_sequence('user_id_seq', '"user"')
+
+    # dataset_class_id_seq
+    logging.info('Updating dataset_class_id_seq...')
+    update_sequence('dataset_class_id_seq', 'dataset_class')
+
+    # dataset_eval_sets_id_seq
+    logging.info('Updating dataset_eval_sets_id_seq...')
+    update_sequence('dataset_eval_sets_id_seq', 'dataset_eval_sets')
```
Comment (alastair) [inline db/dump.py]: Do we have a way of dumping production/private tables yet (e.g. "user"?). If we eventually add it then we'll need to add those tables somewhere too - here or a new method?

File: db/dump.py [hunk 3]
```diff
@@ -20,6 +20,9 @@
 import os
 from sqlalchemy import text
 
+
+logging.basicConfig(level=logging.INFO)
```
Comment (alastair) [inline db/dump.py]: can we use the flask logger here? (I think the management tasks will be running inside an app context, right?)

File: db/dump.py (hunk 2 above)
Comment (paramsingh) [reply db/dump.py]: We don't have a private dump yet, but I updated those sequences in this function too.

File: db/dump.py [hunk 4]
```diff
@@ -273,6 +327,11 @@ def import_db_dump(archive_path):
         connection.close()
 
     pxz.stdout.close()
+    pxz.wait()
+
+    logging.info('Updating sequences...')
```
Comment (paramsingh) [inline db/dump.py]: leaving this rn for consistency, I'll open a seperate PR for converting all these logging calls to use the flask logger.

Comment (paramsingh) [reply db/dump.py]: [AB-366](https://tickets.metabrainz.org/browse/AB-366)

**Pull request merged**